src/machine/machinectl.c

```diff
@@ -82,6 +84,9 @@ static ImportVerify arg_verify = IMPORT_VERIFY_SIGNATURE;
 static const char* arg_format = NULL;
 static const char *arg_uid = NULL;
 static char **arg_setenv = NULL;
+static int arg_addrs = 1;
```

Comment:
I wonder about the type used here. You used a signed type, and use -1 to indicate that all addresses should be shown, permitting 0 as valid number. I wonder about the latter. Should really allow that?

I am tempted to say this should be of type `unsigned`, with `0` indicating "no limit".